app/controllers/application_controller.rb

```diff
@@ -17,6 +17,7 @@ def set_locale
 
   def default_url_options
     { locale: I18n.locale == I18n.default_locale ? nil : I18n.locale }
+    { host: ENV["HOST"] || "localhost:3000" }
```

Comment:
you should add this to the existing Hash (the previous line), instead of creating a new Hash